### PR TITLE
[HOPSWORKS-1665] search

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

File: docs/user_guide/hopsworks/search.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+The search on the navigation bar can be used to search for Projects, Datasets, Feature groups, Features and Training datasets.
+Depending on the page you are in, landing page or project dashboard, the search scope will cover all projects or the specific project you are in.
+
+.. figure:: ../../imgs/search/global_search.png
+    :alt: Global Search
+    :scale: 30%
+    :figclass: align-center
+
+In the case of searching on the landing page, as we can see in the image above, your search will target all items that have not been marked explicitly as being private when they were create. By default all the metadata of items(Projects, Datasets, Feature Groups, Features, Training Datasets) are public and searchable by all users in the cluster. The data however is always private and requires access granting by the owners in order to have access to it. When searching globaly you can narrow your search to the categories: Projects/Datasets/Featurestore(Featuregroup, Features and Training Datasets) by selecting the appropriate drop down option.
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/search.rst:12]:
```suggestion
In the case of searching on the landing page, as we can see in the image above, your search will target all items that have not been marked explicitly as being private when they were created. By default all the metadata of items(Projects, Datasets, Feature Groups, Features, Training Datasets) are public and searchable by all users in the cluster. The data however is always private and requires access granting by the owners in order to allow access. When searching globally you can narrow your search to the categories: Projects/Datasets/Featurestore(Featuregroup, Features and Training Datasets) by selecting the appropriate drop down option.
```

File: docs/user_guide/hopsworks/search.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	* Description
+	* Metadata
+
+.. figure:: ../../imgs/search/global_search_request.png
+    :alt: Request access to data
+    :scale: 30%
+    :align: center
+    :figclass: align-center
+
+When searching globally, if you discover an interesting item, you can click on the paper plane icon in order to send a request to have access to its data. When requesting access you need to provide a project that you want to save this shared item in. If the owner of the item accepts your request, you will now be able to access its contents from the project you selected.
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/search.rst:61]: `request to have access to its data` -> `to be granted access`

File: docs/user_guide/hopsworks/search.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+Search
+========================
+
+The search on the navigation bar can be used to search for Datasets, Feature groups, Features and Training datasets.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/search.rst:4]: Use the active voice, not the passive voice!
"You can search on the navigation bar for Datasets, Feature Groups, Features, and Training datasets.

File: docs/user_guide/hopsworks/search.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Depending on the page you are in, landing page or project dashboard, the search scope will cover all projects or the specific project you are in.
+
+.. figure:: ../../imgs/search/global_search.png
+    :alt: Global Search
+    :scale: 30%
+    :figclass: align-center
+
+In the case of searching on the landing page, as we can see in the image above, your search will target all items that have not been marked explicitly as being private when they were create. By default all the metadata of items(Projects, Datasets, Feature Groups, Features, Training Datasets) are public and searchable by all users in the cluster. The data however is always private and requires access granting by the owners in order to have access to it. When searching globaly you can narrow your search to the categories: Projects/Datasets/Featurestore(Featuregroup, Features and Training Datasets) by selecting the appropriate drop down option.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/search.rst:12]: created

File: docs/user_guide/hopsworks/search.rst [hunk 5]
```diff
@@ -0,0 +1,70 @@
+Search
+========================
+
+The search on the navigation bar can be used to search for Datasets, Feature groups, Features and Training datasets.
+Depending on the page you are in, landing page or project dashboard, the search scope will cover all projects or the specific project you are in.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/search.rst:5]: The search bar behaves differently depending on whether you are on the landing page or inside a project. When you are on the landing page, you can search for datasets and features in all

File: docs/user_guide/hopsworks/search.rst [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+.. figure:: ../../imgs/search/project_search.png
+    :alt: Project Search
+    :scale: 30%
+    :align: center
+    :figclass: align-center
+
+In the case of searching in the project dashboard, your search will target all items that this project has access to (access to their data) - this includes items that were created within this project as well as items that have been shared with this project. When searching within the project you can narrow your search to the categories: Datasets/Featurestore(Featuregroup, Features and Training Datasets).
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/search.rst:20]: Missing a space here before the left parentheses here (same in an earlier sentence above):
Datasets/Featurestore(Featuregroup,

File: docs/user_guide/hopsworks/search.rst [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+	* Description
+	* Metadata
+
+.. figure:: ../../imgs/search/global_search_request.png
+    :alt: Request access to data
+    :scale: 30%
+    :align: center
+    :figclass: align-center
+
+When searching globally, if you discover an interesting item, you can click on the paper plane icon in order to send a request to have access to its data. When requesting access you need to provide a project that you want to save this shared item in. If the owner of the item accepts your request, you will now be able to access its contents from the project you selected.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/search.rst:61]: When you search from the landing page,

File: docs/user_guide/hopsworks/search.rst (hunk 1 above)
Comment (ErmiasG) [reply docs/user_guide/hopsworks/search.rst]: I think allow access will change the meaning of the sentence.

**Pull request merged**